manager/assets/modext/workspace/package.browser.panels.js

```diff
@@ -413,13 +413,15 @@ MODx.panel.PackageBrowserDetails = function(config) {
 				+'<tpl for=".">'
 					+'<div class="selected">'
 						+'<tpl if="screenshot">'
-							+'<a href="{screenshot}" title="'+_('package_preview_view')+'" alt="'+_('package_preview_view')+'" class="lightbox" />'
+							+'<a href="{screenshot}" title="package_preview_view" alt="package_preview_view" class="lightbox">'
```

Comment:
This change is wrong; it removes the title/alt values which need to come from the lexicons.

Reply:
Thanks, I didn't notice that the eslint underscore overlapped the function name

![image](https://user-images.githubusercontent.com/5341208/214638069-c0156141-6554-4555-9abe-a346a7a8b88d.png)